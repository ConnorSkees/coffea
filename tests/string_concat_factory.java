public class string_concat_factory {

    public static void main(String[] args) {}

    public String stringConcatFactory(String x, String y, String z) {
        int a = 2;
        int b = 2;

        String s1 = "foo";
        String s2 = "bar" + 2 + a + 1 + a + 1;
        String s3 = s1 + s2 + x;

        return "";
    }
}