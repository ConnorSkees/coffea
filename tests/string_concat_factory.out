public class string_concat_factory {
  public static void main(String[] arg0) {
    return;
  }
  public String stringConcatFactory(String arg1, String arg2, String arg3) {
    int i4 = 2;
    int i5 = 2;
    String obj6 = "foo";
    String obj7 = (((("bar2" + i4) + "1") + i4) + "1");
    String obj8 = ((obj6 + obj7) + arg1);
    return "";
  }
}
